src/docsfy/storage.py

```
@@ -199,6 +208,33 @@ async def init_db(data_dir: str = "") -> None:
             await db.execute("ALTER TABLE projects_new RENAME TO projects")
             logger.info("Database migration to 5-column PK complete")
 
+        # Migration: add generation_id column
+        try:
+            await db.execute("ALTER TABLE projects ADD COLUMN generation_id TEXT")
+            await db.commit()
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)